Scenes/Tavern/ItemsPanel/ItemsPanel.tscn

```diff
@@ -0,0 +1,95 @@
+[gd_scene load_steps=3 format=2]
+
+[ext_resource path="res://Scenes/Tavern/ItemsPanel/ItemsPanel.gd" type="Script" id=1]
+[ext_resource path="res://Fonts/SHPinsher_16.tres" type="DynamicFont" id=2]
+
+[node name="ItemsPanel" type="Panel"]
+anchor_right = 1.0
+anchor_bottom = 1.0
+script = ExtResource( 1 )
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="MarginContainer" type="MarginContainer" parent="."]
+margin_left = 4.0
+margin_top = 4.0
+margin_right = 556.0
+margin_bottom = 311.0
```

Comment:
Instead of setting `margin_*`, for MarginContainer use `custom_constant/margin_*`. Read more: https://docs.godotengine.org/en/3.2/classes/class_margincontainer.html#description